pallets/asset-manager/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+authors    = ['Manta Network']
+name = "pallet-asset-manager"
+version = "3.1.3"
+edition = "2021"
+homepage   = 'https://manta.network'
+license    = 'GPL-3.0'
+repository = 'https://github.com/Manta-Network/Manta/'
+
+
+[dependencies]
+# codec has to be 2.0.0 for now
```

Comment:
Why 2.0? We're using 2.3.1 in other crates.

Reply:
I get compilation error before. But since now I upgrade to `v0.9.16`, will give it a try now.

Reply:
Oh, I remember now. `2.3.1` works but `3.0.0` doesn't.